documentation/docs/03-loading.md

```diff
@@ -4,6 +4,10 @@ title: Loading
 
 A component that defines a page or a layout can export a `load` function that runs before the component is created. This function runs both during server-side rendering and in the client, and allows you to get data for a page without (for example) showing a loading spinner and fetching data in `onMount`.
 
+> Since `load` can run on the server as well as the client browser, any state changes to a variable that happens on the server will be saved and persisted, and all clients hitting the server-rendered page will be exposed to the already mutated state instead of a clean one.
```

Comment:
I think it's a bit confusing to start this with "Since `load` can run on the server as well as the client browser" because this really only involves the server and so bringing the client into it confused me. "saved and persisted" also makes me think it will persist the value to the client, which it doesn't

Reply:
Initially thought so too, but was trying to preserve the first sentence from

https://github.com/sveltejs/kit/blob/64f749db8ca1e9bb08c38f271d9cd44b31920b83/documentation/docs/03-loading.md#L71

Do you think it would be clearer if we change it back to this?

```suggestion
> It is important to note that `load` can run on the server as well as the client browser, any state changes to a variable that happens on the server will be saved and persisted, and all clients hitting the server-rendered page will be exposed to the already mutated state instead of a clean one.
```

For "saved and persisted" part, I think it was clear with the "that happens on the server" placed right before it. Although, the "server and client" thing was already mentioned in the previous paragraph as well, perhaps we could just omit the sentence before the first comma altogether?

Reply:
Why not just leave that sentence where it was?

Reply:
Feels weird to mention it multiple times throughout the same section, right now twice and would be 3 times if this were added. I do think we can still improve the whole section overall.

Reply:
We wouldn't be mentioning it here if it were moved back though, would we?

Reply:
That's true, I've put them back and placed this at the bottom so it doesn't feel out of place. Users will read the note about the server/client first and we can avoid duplicate mentions.